distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -318,17 +318,17 @@ is_a value typ = if is_same_object value typ then True else
     if typ == Any then True else
         if is_error value then typ == Base.Error else
             case value of
-                Vector.Vector -> typ.is_same_object_as Vector.Vector
-                Array -> typ.is_same_object_as Array
-                Boolean -> if typ == Boolean then True else value == typ
-                Text -> typ == Text
-                Number -> if typ == Number then True else case value of
-                    Integer -> typ == Integer
-                    Decimal -> typ == Decimal
-                Date_Time.Date_Time -> typ.is_same_object_as Date_Time.Date_Time
-                Date.Date      -> typ.is_same_object_as Date.Date
-                Time_Of_Day.Time_Of_Day -> typ.is_same_object_as Time_Of_Day.Time_Of_Day
-                Time_Zone.Time_Zone -> typ.is_same_object_as Time_Zone.Time_Zone
+                _ : Vector.Vector -> typ.is_same_object_as Vector.Vector
+                _ : Array -> typ.is_same_object_as Base.Array
+                _ : Boolean -> if typ.is_same_object_as Base.Boolean then True else value.is_same_object_as typ
+                _ : Text -> typ.is_same_object_as Base.Text
+                _ : Number -> if typ == Base.Number then True else case value of
+                    _ : Integer -> typ == Base.Integer
+                    _ : Decimal -> typ == Base.Decimal
+                _ : Date_Time.Date_Time -> typ.is_same_object_as Date_Time.Date_Time
+                _ : Date.Date      -> typ.is_same_object_as Date.Date
+                _ : Time_Of_Day.Time_Of_Day -> typ.is_same_object_as Time_Of_Day.Time_Of_Day
+                _ : Time_Zone.Time_Zone -> typ.is_same_object_as Time_Zone.Time_Zone
```

Comment:
This is the only complicated change in the `.enso` files in this PR. Everything else is a straightforward rewrite.

Reply:
This is now simplified as I resolved a bug in alias analysis.